<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSeries</class>
 <widget class="QDialog" name="CreateSeries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Series</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QLabel" name="IconLabel">
       <property name="text">
        <string>Icon:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="TitleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="FileAdd">
       <property name="toolTip">
        <string>Sets the icon of the series. Preferred to be 460x690</string>
       </property>
       <property name="accessibleName">
        <string>Image File</string>
       </property>
       <property name="accessibleDescription">
        <string>Click to add an image file, which will serve as the icon.</string>
       </property>
       <property name="text">
        <string>Add Image File...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="TitleEdit">
       <property name="accessibleName">
        <string>Title Box</string>
       </property>
       <property name="accessibleDescription">
        <string>Input the title of the series.</string>
       </property>
       <property name="placeholderText">
        <string>Hunter X Hunter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>TitleEdit</tabstop>
  <tabstop>FileAdd</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateSeries</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateSeries</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
